src/services/builds.ts

```diff
@@ -47,8 +48,12 @@ export const webhookPipelineEventToCompassBuildEvent = (
       build: {
         externalEventSourceId: pipeline.project.id.toString(),
         updateSequenceNumber: lastUpdated.getTime(),
-        displayName: `${pipeline.project.name} pipeline ${pipeline.object_attributes.id}`,
-        description: `Pipeline run ${pipeline.object_attributes.id} for project ${pipeline.project.name}`,
+        displayName: truncateProjectNameString(``, pipeline.project.name, ` pipeline ${pipeline.object_attributes.id}`),
```

Comment:
Out of curiosity, is there a reason why you went with `` to define an empty string? I think the convention is probably to use "" and use `` whenever string interpolation is needed

Reply:
`null` causes issues because I am checking the string length for calculations, but where possible, `` has been changed to "".